### ISSUE TITLE
Touch support

### ISSUE THREAD

**Issue by patik** [danielstocks/react-sortable#15]
This would be fantastic for certain views on mobile devices (i.e. the way you can reorder many UITableViews in iOS). Would it be possible to bind to `touchstart`, `touchmove`, etc as well as the mouse-dragging events?

**Comment by Dharmoslap** [comment]
Working on that

**Comment by oyeanuj** [comment]
@Dharmoslap Is this still on the block?

**Comment by Dharmoslap** [comment]
I was trying find some hack for it, but would be better to wait for mobile browser support. 
If you have some idea, you can check a code and try to implement it.

**Comment by Dharmoslap** [comment]
The problem is that dragOver event e.currentTarget is different for dektop and for touch.

**Comment by oyeanuj** [comment]
@Dharmoslap I see. Could you clarify what are you referring to here - 
> better to wait for mobile browser support

**Comment by danielstocks** [comment]
I think it's more complicated than that unfortunately: here's a discussion regarding what they had to do for react-dnd to make it work: https://github.com/gaearon/react-dnd/issues/16

**Comment by danielstocks** [comment]
TLDR: I think they are not using HTML5 drag and drop API on touch because it was not working very good, so they wrote a custom drag drop API for touch devices.

**Comment by ximex** [comment]
Any updates here?
